crates/firewheel-nodes/src/stream/reader.rs

```diff
@@ -23,44 +22,37 @@ pub const MAX_CHANNELS: usize = 16;
 
 #[derive(Debug, Clone, Copy, PartialEq)]
 pub struct StreamReaderConfig {
```

Comment:
Since this is now used in the `AudioNode` implementation, could we add a `Component` derive here?